extension/src/popup/components/mnemonicPhrase/CheckButton/index.tsx

```diff
@@ -5,12 +5,18 @@ import "./styles.scss";
 
 type CheckButtonProps = {
   onChange: (e: any) => void;
+  onKeyDown?: (e: any) => void;
   wordKey: string;
   word: string;
 };
 
-export const CheckButton = ({ onChange, wordKey, word }: CheckButtonProps) => (
-  <>
+export const CheckButton = ({
+  onChange,
+  onKeyDown,
+  wordKey,
+  word,
+}: CheckButtonProps) => (
+  <span>
```

Comment:
We need to use a wrapping element here in order to use the sibling css selector to make the label outlined when the hidden input is focused. Using a span lets them still render inline and retains the current layout.